content/2_Bronze/Simulation.problems.json

```diff
@@ -78,6 +78,19 @@
       "solutionMetadata": {
         "kind": "internal"
       }
+    },
+    {
+      "uniqueId": "usaco-856",
+      "name": "The Bucket List",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=856",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Simulation"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": 856
```

Comment:
idk if this matters or not, but the usacoId of other problems is a string, while this is a number; maybe it would be better to be consistent?

Reply:
This is a good point, but since the build works properly, that shouldn't stop us from merging this.
If we later decide to standardize the usacoId, it won't be too hard to update it.

Reply:
i already changed the usacoId to a string so all the future problem suggestions should have a string. One reason why we shouldn't leave this one as a number is because when you edit in editor, the schema specifies a string so it will show an error.